src/Common/src/System/Drawing/KnownColorTable.cs

```diff
@@ -56,9 +52,6 @@ private static void InitColorTable()
         {
             int[] values = new int[(unchecked((int)KnownColor.MenuHighlight)) + 1];
 
-#if FEATURE_SYSTEM_EVENTS
-            SystemEvents.UserPreferenceChanging += new UserPreferenceChangingEventHandler(OnUserPreferenceChanging);
-#endif
```

Comment:
@mellinoe, are you ok with removing all of this FEATURE_SYSTEM_EVENTS stuff?  Is there a plan to enable it?  Will removing it mess up Mono's ability to source share?